### ISSUE TITLE
Integration with Gitpod

### ISSUE THREAD

**Issue by mjlbach** [mjlbach/try.nvim#6]
I've been talking with Geoff @ gitpod and would be great to include the CI/infra for using at least one of these containers there.

**Comment by lockejan** [comment]
Ok. Well, if there is enough demand to have such a setup - fine, but for now I don't see it.

Furthermore I think keybindings in vim/nvim/emacs are highly individual.
I wouldn't want to use it if I had to learn new keybindings.

**Comment by mjlbach** [comment]
I don't understand, the keybindings are the same as any of our other containers. 

The feature seems incredibly popular for vscode/emacs users so I don't see why this wouldn't hold for neovim as well.

**Comment by lockejan** [comment]
>I don't understand, the keybindings are the same as any of our other containers.

Yes, you are right. I'm thinking more of having additional plugins which aren't included in the current setup.
Those usually come with keybindings and can result in conflicts, shadowing each others actions.
Of course, this is only a theoretical example, but people have preferences and not everyone favours the current bindings or setup.
That being said, I have nothing against standardising keybindings for the lsp eco system (in nvim) - it's probably a great thing to further invest to, but I'm not sure that the current state (of default keymaps in lsp-config) is widely enough adopted.🤔

>The feature seems incredibly popular for vscode/emacs users so I don't see why this wouldn't hold for neovim as well.

 I don't have any experience working on projects that way, and inside my bubble there hasn't been anyone using such either.
How do you come to that conclusion? (I'm genuinely curious)


Finally, till now the scope of try.nvim has been to help reproducing issues and get an impression of lsp capabilities in neovim.
Using it on a day to day basis in a cloud environment to work on projects is quite a different use case and would need more language based tooling to work effectively on projects IMO.

**Comment by mjlbach** [comment]
I think just having the example gitpod base container is fine, I'm happy to do this myself
